cmd/package-builder/package-builder.go

```diff
@@ -132,7 +142,7 @@ func runMake(args []string) error {
 	}
 
 	currentVersion := version.Version().Version
-	paths, err := packaging.CreatePackages(currentVersion, osqueryVersion, *flHostname, *flEnrollSecret, macPackageSigningKey, *flInsecure, *flInsecureGrpc, *flAutoupdate, *flUpdateChannel, *flIdentifier, *flOmitSecret, *flSystemd, *flCertPins, *flRootPEM)
+	paths, err := packaging.CreatePackages(currentVersion, osqueryVersion, *flHostname, *flEnrollSecret, macPackageSigningKey, *flInsecure, *flInsecureGrpc, *flAutoupdate, *flUpdateChannel, *flControl, *flControlHostname, *flIdentifier, *flOmitSecret, *flSystemd, *flCertPins, *flRootPEM)
```

Comment:
let's refactor this to wrap the options one per line. Maybe it's time for a struct?

Reply:
i think refactoring this to a struct, or better functional options, is out of scope for this PR, but i'll consider revisiting it when there's more time.